software/ever-teams.yml

```
@@ -0,0 +1,14 @@
+name: "Ever Teams"
+website_url: "https://ever.team"
+source_code_url: "https://github.com/ever-co/ever-teams"
+description: "Open Work and Project Management Platform"
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - Nodejs
```

Changed order to reflect install docs - this should be on hold until the docker instructions are complete: https://github.com/ever-co/ever-teams?tab=readme-ov-file#run-in-docker--docker-compose

```suggestion
platforms:
  - Nodejs
  - Docker
```